run-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# create a copy of app, so we can modify as necessary
+cp -a /app /usr/src/out
+
+# add some required files from the template
+cp -a /usr/src/app/.babelrc.js /usr/src/app/helpers /usr/src/out/
+# merge package.json files, prefering whatever is specified in the app
+jq -s '.[0] * .[1]' /usr/src/app/package.json  /app/package.json > /usr/src/out/package.json
```

Comment:
`-s ` is for slurp, which means read the entire thing before applying the operation
`*`  is the multiplier operation, documented as:
> Multiplying two objects will merge them recursively: this works like addition but if both objects contain a value for the same key, and the values are objects, the two are merged with the same strategy.